src/server/zset_family.cc

```diff
@@ -84,7 +84,7 @@ zlexrangespec GetLexRange(bool reverse, const ZSetFamily::LexInterval& li) {
   range.min = GetLexStr(interval.first);
   range.max = GetLexStr(interval.second);
   range.minex = (interval.first.type == ZSetFamily::LexBound::OPEN);
-  range.maxex = (li.second.type == ZSetFamily::LexBound::OPEN);
```

Comment:
Hi @theyueli can you please add unit test to cover this case?

Reply:
@adiholden @romange added test and changed my commit message.